service.py

```diff
@@ -7,11 +7,15 @@
 async def receive_model(request):
     data = await request.post()
     input_file = data['file']
+    type = data['type']
     path = ""
 
     extension = input_file.filename.split('.')[1]
```

Comment:
Burde vi ikke tage den sidste (altså [-1]) for at sikre at det også virker med der har (.) i navnet

Reply:
Fixed